examples/peft_llm_gnn/llm_gnn_model.py

```diff
@@ -89,7 +116,7 @@ def __init__(self, g, node_lm_configs, h_dim, out_dim, num_layers,
                  use_norm = True,   # use normalization or not, default is True
                  alpha_l2norm = 0,
                  lr = 0.001):
-        super(LLMGraphModel, self).__init__()
+        super(GNNLLM_NC, self).__init__()
```

Comment:
GPEFT should also be used here.

Reply:
I intend to do this in a seperate PR that makes the previous node classification uses GPEFT component. Is this okay?

Reply:
if you can remember to do so. it'll be better to do it in the same PR. it'll be more manageable.